openedx/core/djangoapps/celery_utils/management/commands/cleanup_resolved_tasks.py

```diff
@@ -0,0 +1,45 @@
+"""
+Reset persistent grades for learners.
+"""
+from datetime import timedelta
+import logging
+from textwrap import dedent
+
+from django.core.management.base import BaseCommand
+from django.utils.timezone import now
+
+from ...models import FailedTask
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Delete records of FailedTasks that have been resolved
+    """
+    help = dedent(__doc__).strip()
+
+    def add_arguments(self, parser):
+        """
+        Add arguments to the command parser.
+        """
+        parser.add_argument(
+            '--task-name', '-t',
+            default=None,
+            help=u"Restrict cleanup to tasks matching the given task-name.",
+        )
+        parser.add_argument(
+            '--age', '-a',
+            type=int,
+            default=0,
+            help=u"Only delete tasks that have been resolved for at least the specified number of days",
+        )
+
+    def handle(self, *args, **options):
+        tasks = FailedTask.objects.filter(datetime_resolved__lt=now() - timedelta(days=options['age']))
```

Comment:
I added a `--dry-run` option.  

I didn't add the corresponding `--delete` option, because this command doesn't carry a risk of losing user data.  The only task records that will be deleted are ones that are already resolved anyway.  Worst case scenario is we accidentally delete something that *might* help us debug an issue later.  But since the plan is to automate this process, forcing an explicit `--delete` won't really help with that anyway.